jdl/converters/parsed-jdl-to-jdl-object/enum-converter.ts

```diff
@@ -28,7 +28,7 @@ export default { convertEnums };
  * @param {Array} enumerations - parsed JDL enumerations.
  * @return the converted JDLEnums.
  */
-export function convertEnums(enumerations): JDLEnum[] {
+export function convertEnums(enumerations?: ParsedJDLEnum[]): JDLEnum[] {
```

Comment:
If `enumerations` is `undefined`, an error is thrown, so it's not optional.
```suggestion
export function convertEnums(enumerations: ParsedJDLEnum[]): JDLEnum[] {
```